lib/moray_client.js

```diff
@@ -60,7 +59,7 @@ function createClient(options, callback) {
             morayargs.log = options.log.child({
                 component: 'moray-client-' + pnodeUrl.hostname
             });
-            morayargs.crc_mode = options.crc_mode;
```

Comment:
Could add a comment here explaining that this is to support a newer emoray talking to not-yet-updated morays.